src/implementations/deploy/token-registry/token-registry.ts

```diff
@@ -66,7 +65,7 @@ export const deployTokenRegistry = async ({
     implAddress = defaultTokenImplementationContractAddress;
   }
 
-  if ((!deployerContractAddress || !implAddress) && configDeploy) {
+  if (standalone !== false && (!deployerContractAddress || !implAddress)) {
```

Comment:
I just wanted to clarify my understanding if the `configDeploy` flag you had added in your initial commit was only used to determine if the `standalone` flag was true or false, and now your most recent commit (https://github.com/Open-Attestation/open-attestation-cli/pull/271/commits/a8ac97a5b1a9e218cc0b3f3d76e4713d988bb3f7) removes the `configDeploy` because it seemed rather unnecessary and it introduced a bug? So the most recent commit fixes the bug you initially had due to `configDeploy` as it now checks the value of `standalone` directly. Did I understand correctly?